diff --git a/dist/parseSettings/warnAboutTSVersion.js b/dist/parseSettings/warnAboutTSVersion.js
index ca93b3c4d775c5611360372f6d51de82ed4d2d71..24ebb5883e61572a364e0a7aecf33f12786bde11 100644
--- a/dist/parseSettings/warnAboutTSVersion.js
+++ b/dist/parseSettings/warnAboutTSVersion.js
@@ -75,7 +75,7 @@ function warnAboutTSVersion(parseSettings, passedLoggerFn) {
             '\n',
             border,
         ].join('\n');
-        parseSettings.log(versionWarning);
+        // parseSettings.log(versionWarning);
     }
     warnedAboutTSVersion = true;
 }
